src/components/shared/card.js

```diff
@@ -0,0 +1,210 @@
+import Sharpshooter from '../../icons/specializations/sharpshooter.png';
+import Gunner from '../../icons/specializations/gunner.png'
+import Technician from '../../icons/specializations/technician.png'
+import Breadbasket from '../../icons/weapons/talents/Breadbasket.png'
+import ClosePersonal from '../../icons/weapons/talents/Close & Personal.png'
+import Eyeless from '../../icons/weapons/talents/Eyeless.png'
+import FastHands from '../../icons/weapons/talents/Fast Hands.png'
+import Ignited from '../../icons/weapons/talents/Ignited.png'
+import Killer from '../../icons/weapons/talents/Killer.png'
+import LuckyShot from '../../icons/weapons/talents/Lucky Shot.png'
+import Optimist from '../../icons/weapons/talents/Optimist.png'
+import Perpetuation from '../../icons/weapons/talents/Perpetuation.png'
+import Preservation from '../../icons/weapons/talents/Preservation.png'
+import Ranger from '../../icons/weapons/talents/Ranger.png'
+import Reformation from '../../icons/weapons/talents/Reformation.png'
+import Rifleman from '../../icons/weapons/talents/Rifleman.png'
+import Sadist from '../../icons/weapons/talents/Sadist.png'
+import Spike from '../../icons/weapons/talents/Spike.png'
+import SteadyHanded from '../../icons/weapons/talents/Steady Handed.png'
+import Strained from '../../icons/weapons/talents/Strained.png'
+import Vindictive from '../../icons/weapons/talents/Vindictive.png'
+import './selector.css';
+
+const armorIcons = {
+    mask: Mask,
+    backpack: Backpack,
+    vest: Vest,
+    glove: Glove,
+    holster: Holster,
+    kneepads: Kneepads
+};
+
+const weaponIcons = {
+    assault: Assault,
+    lmg: LMG,
+    marksman: Marksman,
+    rifle: Rifle,
+    shotgun: Shotgun,
+    submachine: Submachine
+};
+
+const specializationIcons = {
+    demolitionist: Demolistionist,
+    survivalist: Survivalist,
+    sharpshooter: Sharpshooter,
+    gunner: Gunner,
+    technician: Technician
+};
+
+const weaponTaletsIcons = {
+    breadbasket: Breadbasket,
+    closePersonal: ClosePersonal,
+    eyeless: Eyeless,
+    fasthands: FastHands,
+    ignited: Ignited,
+    killer: Killer,
+    luckyshot: LuckyShot,
+    optimist: Optimist,
+    perpetuation: Perpetuation,
+    preservation: Preservation,
+    ranger: Ranger,
+    reformation: Reformation,
+    rifleman: Rifleman,
+    sadist: Sadist,
+    spike: Spike,
+    steadyHanded: SteadyHanded,
+    strained: Strained,
+    vindictive: Vindictive
+}
+
+function Selector(props) {
+    return (
+        <div className={`holo-card ${props.category}`} id={props.category}>
+            { props.weaponType && (
+                <img className="icon" src={weaponIcons[props.weaponType]} alt={props.category}/>
+            )}
+            
+            { props.armorPiece && (
+                <img className="icon" src={armorIcons[props.armorPiece]} alt={props.category}/>
+            )}
+            { props.specialization && (
+                <>
```

Comment:
whats the purpose of `<>`?

Reply:
<> and </> allow you to return more than one component without adding complexity to the DOM tree. It would essentially be the same as adding a `div` when you want to return more than one element but it does not actually appear in the DOM. It is the shorthand for React Fragments.

https://reactjs.org/docs/fragments.html